osu.Game.Rulesets.Taiko/Difficulty/TaikoPerformanceCalculator.cs

```diff
@@ -35,7 +37,10 @@ protected override PerformanceAttributes CreatePerformanceAttributes(ScoreInfo s
             countMeh = score.Statistics.GetValueOrDefault(HitResult.Meh);
             countMiss = score.Statistics.GetValueOrDefault(HitResult.Miss);
 
-            double multiplier = 1.12; // This is being adjusted to keep the final pp value scaled around what it used to be when changing things
+            // The effectiveMissCount is calculated by gaining a ratio for totalSuccessfulHits and increasing the miss penalty for shorter object counts lower than 1000.
+            effectiveMissCount = Math.Max(1.0, 1000.0 / totalSuccessfulHits) * countMiss;
```

Comment:
Possible div-by-0 (e.g. on drumroll only maps). Needs to be guarded against

Reply:
oops, applied in [3acbcac](https://github.com/ppy/osu/pull/19853/commits/3acbcac4d1a215bb0e17ebb36c3f93309d018cb0)